src/app/(auth)/login/page.tsx

```diff
@@ -22,6 +23,18 @@ export default function Login() {
   const { push } = useRouter();
   const validEmail = (e: string) => e !== '' && isEmail(e);
 
+  useEffect(() => {
+    if (auth && auth.userId) {
+      if (profile && profile.profileData) {
+        if (profile.roles.map(r => r.role).includes('ATTORNEY')) push('/cases');
+        else if (profile.roles.map(r => r.role).includes('LEGAL_FELLOW'))
+          push('/limited-case-assignments');
+        else push('/language-support');
+      }
+      push(CONFIG.onboardingHome);
+    }
+  }, [auth, profile, push]);
```

Comment:
nit on code readability: to reduce nesting, since nothing happens in the case that `auth && auth.userId` is false, we can instead add the reverse check (`!(auth || auth.userId)`) and return if that's true

edge case: let's add a else statement for `push(CONFIG.onboardingHome)`, since `push` itself doesn't return the control of the function (i.e. it could push to `language-support`, then `CONFIG.onboardingHome`).